tiny_dnn/util/serialization_functions.h

```diff
@@ -242,6 +242,90 @@ struct LoadAndConstruct<tiny_dnn::slice_layer> {
   }
 };
 
+template <>
+struct LoadAndConstruct<tiny_dnn::sigmoid_layer> {
+  template <class Archive>
+  static void load_and_construct(
```

Comment:
@edgarriba I am unsure about this. I don't understand yet how cereal wants the design of these functions.